_editorial_policies/02_editor_instructions.md

```diff
@@ -0,0 +1,45 @@
+The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors.
+
+## Pre-review processing
+
+### Lead Editor
+
+The Lead Editor:
+- Handles analysis of presubmission inquiries
+- On article submission, confirms that the article is consistent with the presubmission letter
+- Passes the article and presubmission letter to an appropriate Associate Editor to manage the review process (unless he or she decides to manage the review process directly)
+- Makes a final decision after receiving a recommendation from the Associate Editor and reviewers (see below)
+
+### Associate Editor
+
+Before assigning manuscripts for review, editors have several main tasks:
+- Ensure they do not have a conflict of interest with respect to the work they are to analyze; if they do, [dealing with as dictated by editorial policy](https://livecomsjournal.github.io/policies/editorial_board/).
+- Check to ensure that the manuscript has appropropriate style, grammar, layout, and figure quality to be ready for editing, as in the [instructions for authors](https://livecomsjournal.github.io/authors/policies/). Remember, the journal will not be editing the manuscript, so if you will need to reject the manuscript (for additional revision) because of grammar issues or other stylistic reasons, you should do this *before* sending it for review to avoid wasting the time of the reviewers.
+- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two reviewers, though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.
+
+## Review handling
+
+Once an editor has handled the pre-review steps described above, the review process is largely similar to typical journals. The editor:
+- Contacts suitable reviewers to request reviews, giving them a reasonable (typically three weeks) but not extended amount of time to provide their reviews and noting our conflict of interest policy
+- Handles any potential conflict of interests disclosed by reviewers consistent with (editorial policy](https://livecomsjournal.github.io/policies/editorial_board/)
+- Makes reviewers aware of the [review criteria](https://livecomsjournal.github.io/authors/policies/), including category-specific review criteria
+- Performs a check that the GitHub repository is in order, or ensure the reviewers do so.
+- Ensures reviews are submitted and analyzed in a timely manner (all reviews should be submitted in no more than six weeks), reaching out to remind reviewers as needed and solicit additional reviews if reviewers are too slow or their analysis conflicts
```

Comment:
"(all reviews should be submitted in no more than six weeks)" 
should we add "with a three-week initial target" ?

Reply:
Sounds good.

Reply:
I just noticed that journals are asking (me) for reviews within 10 days or two weeks now.  I really like this idea.  It says, schedule time for this now - don't forget about it.

Reply:
I'm good with 10 days or two weeks also.